app/src/main/res/values-pt-rBR

```diff
@@ -0,0 +1,401 @@
+    <string name="save_state_location">Salvar local do estado</string>
+    <string name="input">Entrada</string>
+    <string name="input_summary">Mapeamento de teclas, layouts, feedback (resposta) tátil</string>
+    <string name="key_mapping">Mapeamento de teclas</string>
+    <string name="key_mapping_summary">Atribuir teclas às entradas de controle</string>
+    <string name="show_soft_input">Mostrar controle de toque</string>
+    <string name="vibrate_on_touch">Vibrar ao tocar</string>
+    <string name="vibration_strength">Força da vibração</string>
+    <string name="soft_input_opacity">Entrada de opacidade suave</string>
+    <string name="cheats">Cheats (Trapaças)</string>
+    <string name="cheats_summary">Ativar &amp; importar códigos de trapaças</string>
+    <string name="enable_cheats">Ativar cheats (trapaças)</string>
+    <string name="cheats_database">Banco de dados de cheats (trapaças) (formato XML)</string>
+    <string name="import_cheats">Importar cheats (trapaças)</string>
+    <string name="import_cheats_summary">Importe cheats (trapaças) de arquivos de bancos de dados para disponibilizá-los. Apenas bancos de dados XML são suportados por enquanto. Qualquer cheat (trapaça) previamente importado será deletado.</string>
+    <string name="rom_shortcut">Atalho da ROM</string>
+
+    <string name="firmware_nickname">Apelido</string>
+    <string name="firmware_message">Mensagem</string>
+    <string name="firmware_language">Idioma</string>
+    <string name="firmware_favourite_colour">Cor favorita</string>
+    <string name="firmware_birthday">Aniversário</string>
+
+    <string name="error_clear_rom_cache">Falha ao limpar o cache da ROM</string>
+    <string name="importing_cheats">Importando cheats (trapaças)…</string>
+    <string name="starting">Iniciando…</string>
+    <string name="move_to_background">Mover para segundo plano</string>
+    <string name="failed_save_cheat_changes">Não foi possível salvar as alterações de trapaça</string>
+    <string name="no_cheats_found">Não foi possível encontrar truques para a ROM atual. Tente importar um banco de dados de cheats (trapaças) diferente.</string>
+    <string name="enabled_cheats">Ativar códigos de trapaça</string>
+    <string name="no_enabled_cheats_for_rom">Você não ativou nenhum código de trapaça para esta ROM</string>
+    <string name="day">Dia</string>
+    <string name="month">Mês</string>
+    <string name="symbol_increase">+</string>
+    <string name="symbol_decrease">-</string>
+
+    <string name="rom_settings">Configurações da ROM</string> <!-- Cadeia de caracteres de acessibilidade. Não se deve usar acrônimos (siglas) -->
+    <string name="label_rom_config_console">Sistema Inicializador</string>
+    <string name="label_rom_config_load_gba_rom">Carregar ROM de GBA</string>
+    <string name="label_rom_config_gba_rom_path">Caminho da ROM de GBA</string>
+    <string name="label_rom_config_gba_save_path">Salvar caminho de GBA</string>
```

Comment:
This translation does not convey accurate information. This is displayed in an option where the user can choose the save file of a GBA ROM. As such, I suggest using "Caminho do arquivo de salvamento GBA".

Reply:
🙂 Alright I'll correct.